### PR TITLE
Update UG according to PE-D issues

### LINKED ISSUE #282: Explain red font usages in docs

### PR DESCRIPTION
**Pull request by cadencjk**
Fixes #282, #264, #238, #233

### CODE REVIEW

**Review by Echomo-Xinyu — APPROVED** [review]
LGTM! There are a few minor resolve conflict issues with my PR and I will fix them on my side after your merge.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -241,23 +240,26 @@ the same convention as [adding a student](#adding-a-student-add).
...
     - Amount paid can only be an integer.
     - Amount paid must be non-negative.
 
-5. Amount Owed:
+5. Amount Owed (`owed/`):
     - Amount owed can only be an integer.
     - Amount owed must be non-negative.
 
-6. Rates per Class:
+<div markdown="span" class="alert alert-success">ℹ **Note:** Notice how the student's name changed to red? This is because the student now owes money!
```
Comment (sjoann) [inline docs/UserGuide.md:251]: class="alert alert-info" for the correct background colour

File: docs/UserGuide.md [hunk 2]
```diff
@@ -296,14 +298,20 @@ The student's next class will be set a week later at the same time, provided if
...
+
+<div markdown="span" class="alert alert-success">💡 **Tip:** If you want to unmark a student, you may do so via the `undo` command.
+</div>
 
 Example:
 - `mark 2` marks the 2nd student in the Schedule panel.
 
 ![UiMark](images/UG-screenshots/UiMark.png)
 
+<div markdown="span" class="alert alert-success">ℹ **Note:** Notice how the student's name changed to red? This is because the student now owes money!
```
Comment (sjoann) [inline docs/UserGuide.md:312]: alert alert-info

**Review by sjoann — APPROVED** [review]
LGTM ! just need to change the class attribute of div tag accordingly

**Pull request merged**